backend/app/main.py

```
@@ -0,0 +1,126 @@
+
+    # 3. Health Check
+    @app.get("/health", tags=["System"])
+    async def health_check():
+        """Health check endpoint with database connectivity test."""
+        try:
+            # Test database connection
+            db = SessionLocal()
+            try:
+                db.execute(text("SELECT 1"))
+                db_status = "connected"
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)